net-misc/rsync/rsync-9999.ebuild

```diff
@@ -56,11 +56,9 @@ python_check_deps() {
 
 src_prepare() {
 	default
-	if [[ "${PV}" == *9999 ]] ; then
```

Comment:
Please do not remove this condition in the live ebuild. When we copy this ebuild for the next version bump, we do not want to call the autotools functions unnecessarily.

Reply:
Sure, i have removed the changes from 9999 ebuild.